packages-content-model/roosterjs-content-model-api/lib/modelApi/block/setModelAlignment.ts

```diff
@@ -56,15 +56,24 @@ export function setModelAlignment(
     );
 
     paragraphOrListItemOrTable.forEach(({ block }) => {
-        const newAligment = ResultMap[alignment][block.format.direction == 'rtl' ? 'rtl' : 'ltr'];
-        if (block.blockType === 'Table') {
+        const newAlignment =
+            alignment === 'justify'
```

Comment:
Please update the value map at line 27, so that no need to have a separate check for 'justify' here.

We should make the code follow the same flow as much as possible.